.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+	extends: ['./node_modules/@balena/lint/config/.eslintrc.js'],
+	parserOptions: {
+		project: 'tsconfig.dev.json',
+		sourceType: 'module',
+	},
+	root: true,
+	env: {
+		jest: true
+	},
+	overrides: [
+		{
+			files: ["*.js"],
+			rules: {
+				"@typescript-eslint/no-var-requires": "off"
```

Comment:
This is the only part I don't like about this PR. It was either that or converting all the tests to TS, but I got slightly blocked trying to convert as using native imports on them would require to use esm loader which was not working properly for browser tests. @thgreasi let me know if you have other solution or ideas here